### PR TITLE
object

### PR DESCRIPTION
**Pull request by avagyan98**

### CODE REVIEW

File: object.js [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+
+// 2. Check for emptiness
+
+function isEmpty (obj, key='key'){
+    console.log(key)
+    if(key in obj) {
+        return false
+    }
+    return true
+}
```
Comment (robhov97) [inline object.js]: it will not work correctly

**Review by robhov97 — CHANGES_REQUESTED** [review]
possibly artagratsa

File: object.js (hunk 1 above)
Comment (avagyan98) [reply object.js]: changed

File: object.js [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+
+delete user.name;
+
+console.log(user);
+
+
+// 2. Check for emptiness
+
+function isEmpty (obj){
+    return !Object.keys(obj)[0];
```
Comment (robhov97) [inline object.js]: if you went in this, just check the length of the array and not the first element

**Review by robhov97 — CHANGES_REQUESTED** [review]

File: object.js (hunk 2 above)
Comment (avagyan98) [reply object.js]: done

**Review by robhov97 — APPROVED** [review]

**Pull request merged**